Add admin_ids to meeting.create, add oml check to has_cml.

reiterl opened the pull request:
Update tests.
Two things are done in this pr. The oml can_manage_organization is checked in the has_committee_management_level, too.
Add admins setting to meeting.create.

jsangmeister commented on openslides_backend/action/actions/meeting/create.py:
```diff
@@ -149,14 +164,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                     "group_$_ids": {str(instance["id"]): [fqid_default_group.id]},
                 }
                 for user_id in user_ids
-                if user_id != self.user_id
```
You probably still have to filter out all users which you added as admins, because else you will overwrite the admin group and all admins will end up in the default group. See also comment regarding the test above

reiterl replied:
Filter out all admins here. The admins should go into the admin group and not into the default group.

jsangmeister reviewed (changes requested):
Please also add a permission test that you can now execute `meeting.create` with the correct CML.

reiterl commented:
Add some tests for the permissions and admin_ids.

reiterl commented on tests/system/action/meeting/test_create.py:
```diff
@@ -156,69 +149,36 @@ def test_check_action_data_fields(self) -> None:
-            "user/1", {f"group_${meeting['id']}_ids": [admin_group_id]}
-        )
 
-    def test_create_with_request_user(self) -> None:
```
Readded a same user in users and admins test.

jsangmeister commented on tests/system/action/meeting/test_create.py:
```diff
@@ -244,3 +252,44 @@ def test_create_permissions(self) -> None:
+            "user/2", {f"group_${meeting['id']}_ids": [admin_group_id]}
+        )
+
+    def test_create_with_admin_ids_and_no_permissions(self) -> None:
```
Isn't this test redundant with `test_create_no_permissions` above?

reiterl replied:
It has a slightly different payload, but I think we can remove it.

jsangmeister reviewed (changes requested):
A test where the user has OML `can_manage_organization` is still missing

reiterl commented:
I added a test _test_create_with_admin_ids_and_permissions_oml,_ where the user 1 has oml can_manage_organization.
I removed the surplus test _test_create_with_admin_ids_and_no_permissions_ .
